hummingbot/connector/exchange/bitmart/bitmart_exchange.py

```diff
@@ -641,6 +641,9 @@ async def _update_order_status(self):
             responses = await safe_gather(*tasks, return_exceptions=True)
             for response in responses:
                 if isinstance(response, Exception):
+                    if hasattr(response, "message"):
+                        if response.message == "order not exist":
```

Comment:
Is this a real message? I can't find it in the Bitmart API documentation.

Reply:
Its a message I saw when trying to remove and/or retrieve the order. Skipping removing items that didnt exist seemed legitimate to me.